module/VuFind/src/VuFind/Db/Entity/Search.php

```diff
@@ -127,6 +131,13 @@ class Search implements SearchEntityInterface
      */
     protected $searchObject;
 
+    /**
+     * Normalized search object after loading.
+     *
+     * @var \VuFind\Search\Minified|null
+     */
+    private $deserializedSearchObject = null;
```

Comment:
We generally favor protected over private to improve extensibility:

```suggestion
    protected $deserializedSearchObject = null;
```